setup.cfg

```diff
@@ -0,0 +1,23 @@
+[pycodestyle]
+exclude = .eggs,ENV,build,docs/conf.py,venv
+
+[yala]
+radon mi args = --min C
+pylint args = --disable=too-many-locals,too-few-public-methods,too-many-instance-attributes,too-many-arguments,inconsistent-return-statements,unnecessary-pass,too-many-public-methods,redefined-outer-name,unnecessary-lambda,missing-timeout,import-error,no-name-in-module,attribute-defined-outside-init
```

Comment:
Now with the running `tox` give a second look at all these disables. I could be wrong but I think you do not need `missing-timeout`